fs/flail.py

```diff
@@ -42,7 +42,7 @@ def setUp(self):
         archive.extract(self.get_data("flail-0.2.0.tar.gz"), self.workdir)
         self.build_dir = os.path.join(self.workdir, 'flail-0.2.0')
         os.chdir(self.build_dir)
-        process.system("sed -i 's/.*-lm.*/& -lm' Makefile")
+        process.system("sed -i 's/.*-lm.*/& -lm/' Makefile")
```

Comment:
@PraveenPenguin can this be made into a python file operation instead of bash

Reply:
@PraveenPenguin any updates ? Did you miss this ?

Reply:
ah sorry i miss tat, will send PR soon